### PR TITLE
adding leftIcon and leftAvatar to dropdown chips

### PR DESCRIPTION
**Pull request by maxiozer**
#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ ] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [Monday Design System](https://design.monday.com/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

File: src/components/Dropdown/components/ValueContainer/ValueContainer.jsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ export default function Container({ children, selectProps, ...otherProps }) {
               e.stopPropagation();
             }}
             readOnly={withMandatoryDefaultOptions && option.isMandatory}
+            leftAvatar={option.leftAvatar}
```
Comment (orrgottlieb) [inline src/components/Dropdown/components/ValueContainer/ValueContainer.jsx:42]: it is too specific, can you pass your own value container ?

**Review by orrgottlieb — CHANGES_REQUESTED** [review]

**Review by orrgottlieb — APPROVED** [review]

**Review by hadasfa — CHANGES_REQUESTED** [review]
Please add a snapshot test for avatar and icon and move the example to different story :)

**Comment by maxiozer** [comment]
image

**Review by hadasfa — APPROVED** [review]
Awesome

**Pull request merged**